docs/framework/wcf/feature-details/system-web-routing-integration.md

```diff
@@ -9,7 +9,7 @@ When hosting a Windows Communication Foundation (WCF) service in Internet Inform
 ## Using System.Web.Routing Integration  
  To use the <xref:System.Web.Routing> integration feature, you use the <xref:System.ServiceModel.Activation.ServiceRoute> class to create one or more routes and add them to the <xref:System.Web.Routing.RouteTable> in a Global.asax file. These routes specify the relative URIs that the service responds to. The following example shows how to do this.  
   
-```  
+```javascript  
```

Comment:
There's JavaScript in the <script> element, but not sure the whole thing should be labeled JavaScript.

Reply:
I changed this to "aspx-csharp" because it is an aspx file. If that's not right, then we should remove the language id.